src/main/java/io/r2dbc/postgresql/PostgresqlConnectionFactoryProvider.java

```diff
@@ -72,6 +79,16 @@ public PostgresqlConnectionFactory create(ConnectionFactoryOptions connectionFac
             builder.port(port);
         }
 
+        Boolean ssl = connectionFactoryOptions.getValue(SSL);
+        if (ssl != null) {
+            builder.ssl(ssl);
+        }
+
+        SslProvider sslProvider = connectionFactoryOptions.getValue(SSL_PROVIDER);
```

Comment:
PostgresConnectionFactoryProvider is typically used through URL parsing. Let's omit `SslProvider` for now and add later something for certificate and verification mode properties.